sky/packages/sky/lib/src/fn3/scrollable.dart

```diff
@@ -22,6 +22,7 @@ const double _kMinFlingVelocity = -kMaxFlingVelocity * _kMillisecondsPerSecond;
 const double _kMaxFlingVelocity = kMaxFlingVelocity * _kMillisecondsPerSecond;
 
 typedef void ScrollListener(double scrollOffset);
+typedef double ToSnapOffset(double scrollOffset);
```

Comment:
ToSnapOffset sounds like a subclass of Offset.
Maybe call this ToSnapOffsetCallback or some such?